README.md

```diff
@@ -50,9 +45,6 @@ class BuildSchemaSubscriber implements EventSubscriberInterface
         ];
     }
 
-    /**
-     * @param \EzSystems\DoctrineSchema\API\Builder\SchemaBuilderEvent $event
-     */
     public function onBuildSchema(SchemaBuilderEvent $event)
```

Comment:
Since `importSchema` received returned type, shouldn't this method do so as well?

Reply:
Ah, yes, I shouldn't modify return type at all, it's a bit out of scope. Can we postpone it till later?

Reply:
I mean you've added it to the other method :smile: So I'd either add the types, or not modify that at all.

But :+1: nonetheless.